.github/workflows/fedora33/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM fedora:33
+
+# install dependencies
+RUN dnf -y install \
+      https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm \
+      https://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm
+
+RUN dnf -y install \
+      Xvfb \
+      cairo-devel \
+      cairo-gobject-devel \
+      dbus-devel \
+      dbus-x11 \
+      glib2-devel \
+      gobject-introspection-devel \
+      gstreamer1 \
+      gstreamer1-libav \
+      gstreamer1-plugins-{bad-\*,good-\*,base} \
+      gtk3 \
+      poetry \
+      python3-devel \
+      python3-feedparser \
```

Comment:
I think most of these we can / should get from Poetry?

Reply:
I believe I installed all of those packages for specific errors I got when I was trying to run **poetry install**

I'll double check though

Reply:
You were right.

I'm cut down on the imports by alot now!

I tested this via Docker image directly, with the **nektos/act** tool, and it's passing in GH Actions,
so hopefully these tests will prove to not be flaky